scripts/generate_schemas.py

```diff
@@ -227,6 +228,24 @@ def write(self):
                 c = classes[name]
                 c.write(f)
 
+            # can't just use pprint library because
```

Comment:
Hadn't heard of `pprint` before, looks really useful! What's the issue with pep8?